app.rb

```diff
@@ -0,0 +1,103 @@
+require './classes/person'
+require './classes/student'
+require './classes/classroom'
+require './classes/teacher'
+require './classes/capitalize_decorator'
+require './classes/trimmer_decorator'
+require './classes/book'
+require './classes/rental'
+
+class App
+  def initialize
+    @people = []
+    @books = []
+    @rentals = []
+  end
+
+  def options
+    puts 'Please enter the number of the option:'
+    puts '1 - List all books'
+    puts '2 - List all people'
+    puts '3 - Create a person'
+    puts '4 - Create a book'
+    puts '5 - Create a rental'
+    puts '6 - List all rentals for a given person id'
+    puts '7 - Exit'
+  end
```

Comment:
- [ ] Kindly consider re-rendering the `options` after each successful operation performed as shown in the video given in the project requirements. For `example` after creating a book re-render the `options` and It should be repeated for all operations from 1 - 6.